morango/migrations/0007_auto_20171018_1615.py

```diff
@@ -113,7 +113,7 @@ class Migration(migrations.Migration):
         migrations.AddField(
             model_name='syncsession',
             name='connection_kind',
-            field=models.CharField(choices=[('network', 'Network'), ('disk', 'Disk')], default=datetime.datetime(2017, 10, 18, 21, 15, 12, 858664, tzinfo=utc), max_length=10),
+            field=models.CharField(choices=[('network', 'Network'), ('disk', 'Disk')], default='', max_length=10),
```

Comment:
Generally very unrecommended to modify an existing migration. But the migration hasn't been shipped anywhere previously, has it? In Kolibri?

Reply:
(and changing a default like this shouldn't be a big issue)

Reply:
yes to the 1st question and yeah thats what I thought about the default as well.